codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPProtocolCustomizations.kt

```diff
@@ -38,7 +38,6 @@ abstract class AWSHTTPProtocolCustomizations : DefaultHTTPProtocolCustomizations
         if (config.serviceSpecificConfigProperties().any { it.memberName == "accountIdEndpointMode" }) {
             writer.write("  .withAccountIDEndpointMode(value: config.accountIdEndpointMode)")
         }
-        writer.write("  .withIdentityResolver(value: config.awsCredentialIdentityResolver, schemeID: \$S)", "aws.auth#sigv4")
```

Comment:
This identity resolver is now set in smithy-swift, since sigv4 is part of base / non-AWS Smithy.